src/data/nat/factorial.lean

```diff
@@ -143,4 +143,37 @@ begin
   { exact add_factorial_succ_le_factorial_add_succ i h }
 end
 
+section desc_fac
+
+-- is there some way to tell only `norm_num` to expand this out, but not `simp`?
+/-- desc_fac n k = (n + k)! / n!, but implemented in a recursive way for calculation. -/
```

Comment:
Could you add a sentence to the doc-string explaining the relationship with `pochhammer`? (Perhaps even a sentence at `pochhammer`, too?)